packages/thirdweb/src/chains/chain-definitions/linea-sepolia.ts

```diff
@@ -0,0 +1,15 @@
+import { defineChain } from "../utils.js";
+
+export const lineaSepolia = /* @__PURE__ */ defineChain({
+  id: 84532,
```

Comment:
this chain id is "base sepolia"

```suggestion
  id: 59141,
```